examples/defer/defer.go

```diff
@@ -40,5 +40,10 @@ func writeFile(f *os.File) {
 
 func closeFile(f *os.File) {
 	fmt.Println("closing")
-	f.Close()
+	err := f.Close()
+	// It is important to check for errors when closing a file, even in a deferred function.
```

Comment:
Please run `tools/build` - there's a chance this will overflow the maximum column width and you'll have to reflow this comment.

Reply:
tools/build has terminated without errors